frontend/redocly.yaml

```diff
@@ -0,0 +1,9 @@
+apis:
+  bn@v2:
+    root: https://dharmamitra.org/api-db/openapi.json
+    x-openapi-ts:
+      output: ./src/codegen/api/v2.ts
+  bn@v2-proto:
```

Comment:
we can probably remove one of these now, I think @sebastian-nehrdich took it down.